.github/workflows/circuit-t3rn-collator-release.yml

```diff
@@ -0,0 +1,181 @@
+name: Circuit t3rn Rococo Collator Release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
```

Comment:
I think this will also release on *-.rc* tags, this should be more specific to something like `v.N+.+.`

Reply:
heres a regex, tested on https://regex101.com/r/YTbC7V/1

`/v\d+\.\d+\.\d+/g`